src/main/java/org/spongepowered/mod/mixin/core/server/MixinServerConfigurationManager.java

```diff
@@ -191,16 +205,6 @@ public void initializeConnectionToPlayer(NetworkManager netManager, EntityPlayer
         playerIn.getStatFile().func_150884_b(playerIn);
         this.func_96456_a((ServerScoreboard) worldserver.getScoreboard(), playerIn);
         this.mcServer.refreshStatusNextTick();
-        ChatComponentTranslation chatcomponenttranslation;
-
-        if (!playerIn.getCommandSenderName().equalsIgnoreCase(s)) {
-            chatcomponenttranslation = new ChatComponentTranslation("multiplayer.player.joined.renamed", new Object[] {playerIn.getDisplayName(), s});
-        } else {
-            chatcomponenttranslation = new ChatComponentTranslation("multiplayer.player.joined", new Object[] {playerIn.getDisplayName()});
-        }
-
-        chatcomponenttranslation.getChatStyle().setColor(EnumChatFormatting.YELLOW);
-        this.sendChatMsg(chatcomponenttranslation);
```

Comment:
If you keep everything except this here (comment this line out) you can minimize the diff, try to keep the diff between the original version and our overwrite as small as possible, that will simplify updating